src/components/TransactionItem/Details/Details.js

```diff
@@ -1,62 +1,146 @@
+
+    return (
+      <div className={classNames('transaction-item-details', isActive && '-active')}>
+        <div className='item'>
+          <div className='label'>
+            <JText value='Tx hash' color='gray' />
+          </div>
+          <div className='value'>
+            <a
+              href={getTxLink(data.hash)}
+              target='_blank'
+              className='link'
+              rel='noopener noreferrer'
+            >
+              {data.hash}
+            </a>
+          </div>
         </div>
-        <div className='value'>
-          <a
-            href={getTxLink(data.transactionHash)}
-            target='_blank'
-            className='link'
-            rel='noopener noreferrer'
-          >
-            {data.transactionHash}
-          </a>
+        <div className='item -small-width'>
+          <div className='label'>
+            <JText value='From address' color='gray' />
+          </div>
+          <div className='value'>
+            <a
+              href={getTxLink(data.from)}
+              target='_blank'
+              className='link'
+              rel='noopener noreferrer'
+            >
+              {data.from}
+            </a>
+          </div>
         </div>
-      </div>
-      <div className='item'>
-        <div className='label'>
-          <JText value='Fee' color='gray' />
+        <div className='item -small-width'>
+          <div className='label'>
+            <JText value='To address' color='gray' />
+          </div>
+          <div className='value'>
+            <a
+              href={getTxLink(data.to)}
+              target='_blank'
+              className='link'
+              rel='noopener noreferrer'
+            >
+              {data.to}
+            </a>
+          </div>
         </div>
-        <div className='value'>
-          <JText value={`${data.fee} ETH`} color='gray' />
+        <div className='item'>
+          <div className='label'>
+            <JText value='Fee' color='gray' />
+          </div>
+          <div className='value'>
+            <JText value={`${fee} ${assetSymbol}`} color='gray' />
+          </div>
         </div>
-      </div>
-      <div className='actions'>
-        <div className='repeat'>
-          <JRaisedButton
-            onClick={handle(repeat)(assoc('symbol', assetSymbol)(data))}
-            color='white'
-            label='Repeat'
-            iconColor='blue'
-            iconName='repeat'
-            labelColor='blue'
-          />
+        <div className='actions'>
+          <div className='action' onClick={handle(repeat)(assoc('symbol', assetSymbol)(data))}>
+            <div className='icon'>
+              <JIcon size='medium' color='gray' name='repeat' />
+            </div>
+            <div className='text'>
+              <JText value='Repeat payment' color='gray' size='normal' />
+            </div>
+          </div>
+          <div className='action' onClick={addFavorite}>
```

Comment:
We should use one of our button component here, may be JRoundButton (as i remember)

Reply:
JFlatButton